### PR TITLE
MAUI-846428-Prepare UG documentation for show and hide tooltip 

### PR DESCRIPTION
**Pull request by SuryaKaran2143**
**Description**

Included the user guide content for show and hide tooltip methods.

### CODE REVIEW

File: MAUI/Cartesian-Charts/Tooltip.md [hunk 1]
```diff
@@ -221,3 +221,72 @@ this.Content = chart;
...
+        public MainPage()
+        {
+            InitializeComponent();
+        }
+
+        private void Button_Clicked(object sender, EventArgs e)
+        {
+            //pointX - determines the x position of tooltip, pointY - determines the y position of tooltip and bool value determines whether the tooltip should be animated while displaying.
+            tooltip.Show(pointX,pointY,true);
+        }
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Tooltip.md:267]: tooltip.Show(pointX,` `pointY,` `true);

Code alignment not good.

File: MAUI/Cartesian-Charts/Tooltip.md [hunk 2]
```diff
@@ -221,3 +221,72 @@ this.Content = chart;
...
+
+The [Hide]() method is used to hide the tooltip programmatically.
+
+{% tabs %}
+
+{% highlight c# %}
+
+        //The argument determines whether the tooltip should be animated while hiding.
+
+        tooltip.Hide(true);
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Tooltip.md:287]: we not have parameter for hide member. 

Not checked the APIs properly before preparing the UG?

File: MAUI/Cartesian-Charts/Tooltip.md (hunk 1 above)
Comment (Saravanan-Madhesh) [reply MAUI/Cartesian-Charts/Tooltip.md:267]: Remove unwanted code, mainpage and its constructor.

**Review by Saravanan-Madhesh — CHANGES_REQUESTED** [review]

File: MAUI/Cartesian-Charts/Tooltip.md (hunk 1 above)
Comment (SuryaKaran2143) [reply MAUI/Cartesian-Charts/Tooltip.md:260]: I corrected the code alignment
I removed the mainPage and its contructor
I only include the private method only

File: MAUI/Cartesian-Charts/Tooltip.md (hunk 2 above)
Comment (SuryaKaran2143) [reply MAUI/Cartesian-Charts/Tooltip.md]: I remove the parameter from the Hide method.
Apologies for this mistake .

**Review by Saravanan-Madhesh — APPROVED** [review]
The changes are fine.

**Pull request merged**